casetokens.php

```diff
@@ -228,6 +228,11 @@ function casetokens_civicrm_tokens(&$tokens) {
       catch (Throwable $ex) {
       }
     }
+    //adding tokens for client case role
+    foreach ($allFields as $key =>$field) {
+      $tokens['case_roles']["case_roles.client_{$key}"] =
+        "Case Client". ' - ' . ts(ucwords($field));
+    }
```

Comment:
yeah I tried for the same earlier but the issue is that it makes an api call to fetch that relation type and then use that name so it will require some if else checks and make code uglier also it does not increase the loop in actual.